### ISSUE TITLE
Include MSVC++ 2012 or higher in the dependencies check

### ISSUE THREAD

**Issue by LipkeGu** [OpenRA/OpenRA#8379]
since we have the check for MSVC++ 2010 it is unable to compile openra... i have installed VS 2013 =(
which installs Visual C++ runtimes 2012 and freetype6 compiled also fine with this... =(

now it fails with every Version =(

**Comment by penev92** [comment]
I did not know it can do that.
Pinging @Mailaender  and @Robmaister to confirm what versions it can use (for what versions the script needs to check).

**Comment by LipkeGu** [comment]
on my laptop is only visual studio 2013 (x86) installed (without 2010) and it compiles fine (without errors...

**Comment by penev92** [comment]
By the way this can't be breaking anything. It just tells you that you should get MSVC++ 2010.
It only does that *after* all dependencies are fetched, and has nothing to do with the build process.

**Comment by LipkeGu** [comment]
no it fails direktly when i run ```make all``` on my laptop there is no Visual C++ runtimes installed and ```make all``` says no errors

**Comment by penev92** [comment]
OK, what error do you get and when?

**Comment by LipkeGu** [comment]
I fetched a few days out the bleed branch -> works

Today: i fetched out again: works not

thats what i did today:

1. ```fetched & rebased```
2. ```make clean``` & ````make dependencies``` (got red warning) & ```make all ``` -> 7 warnings and 108 Errors =(

on my laptop i repeated these steps... and build was successfully

but on my laptop VS is using: C:\Program Files\Microsoft Visual Studio 12.0\VC\lib\msvcrt.lib

**Comment by LipkeGu** [comment]
CoreResGen seems to be depending on the Windows SDK :/
```CoreResGen:
  "C:\Program Files (x86)\Microsoft SDKs\Windows\v8.0A\bin\NETFX 4.0 Tools\resgen.exe" /useSourcePath /r:D:\Projekte\OpenRA\thirdparty\download\Eluant.dll /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\mscorlib.dll" /r:D:\Projekte\OpenRA\OpenRA.Game.exe /r:D:\Projekte\OpenRA\OpenRA.Mods.Common\bin\Debug\OpenRA.Mods.Common.dll /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Core.dll" /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Data.DataSetExtensions.dll" /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Data.dll" /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.dll" /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Drawing.dll" /r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Windows.Forms.dll" /r:"C:\```

**Comment by LipkeGu** [comment]
I made a Clean installation of Windows 7 which fullfills the Requiments to build openra and installed only MSVC++ Runtimes 2013 (x86) and i can also compile OpenRA but with Warning.

Version 2013 (x86) only seems also to work

**Comment by Mailaender** [comment]
I suggest you simply check for VS C++ runtimes > 2010, too then in the build scripts, but I don't know anything about Windows development and can't really test this.

**Comment by LipkeGu** [comment]
#8382

**Comment by Robmaister** [comment]
Sounds like this is pretty much resolved with PR #8382 but if you want SharpFont.Dependencies to ship with a special configuration let me know and I'll try to set it up (VC 6/7 seem to be impossible to get running on Windows 8.1). It would then only require changing your thirdparty fetch scripts to point to the correct folder.
